common/core/package.json

```diff
@@ -16,7 +16,8 @@
     "mocha": "^3.0.1",
     "sinon": "^3.2.1",
     "tslint": "^4.5.1",
-    "typescript": "2.0.6"
+    "typescript": "2.0.6",
+    "uuid": "^2.0.1"
```

Comment:
>"uuid": "^2.0.1" [](start = 4, length = 16)

why?

Reply:
mistake

---
In reply to: [140601320](https://github.com/Azure/azure-iot-sdk-node/pull/88#discussion_r140601320) [](ancestors = 140601320)